doc/gov/chapters/chapter04.md

```diff
@@ -4,66 +4,106 @@
   * [4.2.1 Length of Terms](#4.2.1)
   * [4.2.2 Representation](#4.2.2)
+  * [4.2.3 Nomination](#4.2.4)
+  * [4.2.4 Selection](#4.2.4)
+  * [4.2.5 Interim Leads/Co-Leads](#4.2.5)
 * [4.3 Technical Steering Committee Lead/Co-Lead Nomination and Selection](#4.3)
-  * [4.3.1 Length of Terms](#4.3.1)
-  * [4.3.2 Representation](#4.3.2)
-  * [4.3.3 Membership and voting rights](#4.3.3)
 * [4.4 Governance Committee Lead/Co-Lead Nomination and Selection](#4.4)
-  * [4.4.1 Length of Terms](#4.4.1)
-  * [4.4.2 Representation](#4.4.2)
-  * [4.4.3 membership and voting rights](#4.4.3)
-* [4.5 Committee Structure](#4.5)
-* [4.6 Limits and Caps](#4.6)
-* [4.7 Tracking contributions](#4.7)
+* [4.5 Membership and voting rights](#4.5)
+* [4.6 Committee Structure](#4.6)
+* [4.7 Limits and Caps](#4.7)
+* [4.8 Tracking contributions](#4.8)
 
 <a name="4.1"></a>
 ## 4.1 Introduction
 
+The CNTT is structured into Workstreams so it can scale and accommodate participation from the wider community. The CNTT is open to participation by any individual or organization that agrees to comply with it's ToR (each such participant, a “Participant”). Each of the Workstreams within CNTT has one Lead and up to two Co-Leads. Leads and Co-Leads for each Workstream are selected by the community itself through nomination and voting processes. This document explains the CNTT policies and process in details regarding nomination and selection.
+
 <a name="4.2"></a>
 ## 4.2 Workstreams Lead/Co-Lead Nomination and Selection
 
+For each Workstream, following leadership positions exist:
+- Lead.
+- Up to two Co-leads. 
+
 <a name="4.2.1"></a>
 ### 4.2.1 Length of Terms
 
+The length of term for all CNTT leadership roles are **12** calender months.
+
 <a name="4.2.2"></a>
 ### 4.2.2 Representation
 
+Any Workstream Leads or Co-Leads positions can be held by any community individuals.
+
+<a name="4.2.3"></a>
+### 4.2.3 Nomination
+
+- Any individual can nominate themselves or someone else for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
+- An individual can self-nominate themselves to as many positions as they like.
+
+<a name="4.2.4"></a>
+### 4.2.4 Selection
+
+There wil be a nomination window of which individuals an nominate others or nominate themselves for a particular position. The start and end date of the nomination window will be determined by Technical or Governance Steering Leads(depending on the nature of the Workstream).
+
+Once the nomination period expires for a particular Workstream, the selection process will start as follows:
+
+- Selection process for Leads positions and Co-Leads will occur separately.
+- No voting will be required if number of nominations received for a particular position is less than or equal to the number of available vacancies. 
+- Online Voting process will take place to select candidate(s) from the nominee list for any given position.
```

Comment:
Suggest this document defines the voting process/tool.
e.g. Doodle? [Condorcet](https://civs.cs.cornell.edu/)?

Reply:
added some clarification: https://github.com/cntt-n/CNTT/pull/896/commits/8f4687eccba56ebe7a43520f7b824efd44bfa0ca

Reply:
Thanks - resolved.